### ISSUE TITLE
Adding yaml and initiating travis to automatically check JSON format for every pull request.

### ISSUE THREAD

**Issue by ay2306** [vinitshahdeo/inspirational-quotes#816]
There are many pending pull requests. We can initiate Travis to directly check if PR has the correct JSON format or not.

**Comment by kusha** [comment]
@ay2306 I've created #818 to address this request
The benefit of Github Actions for this use case as there is no need to set up anything outside of Github.
The drawback is that I think it won't run for existing PRs as there is a need to rebase it on top of `.github/workflows` config.
It is not a problem to add `jsonschema` there to validate data.json content (list of dicts w/ specific fields)

**Comment by ay2306** [comment]
@kusha Yes, I am looking into it since yesterday and I couldn't find a way to run tests for the previous PRs.

**Comment by MattRudy** [comment]
is this still open? I don't see any active actions running on this repository and I've set this up before - I can get it setup to run on-demand for PRs.

**Comment by johnemurphy1** [comment]
> is this still open? I don't see any active actions running on this repository and I've set this up before - I can get it setup to run on-demand for PRs.

did this get assigned?
